grails-app/services/au/org/emii/portal/WpsService.groovy

```diff
@@ -49,15 +49,26 @@ class WpsService extends AsyncDownloadService {
 
         params.expirationPeriod = _getExpirationPeriod()
         params.jobReportUrl = _getJobReportUrl(params)
-        params.contactEmail = grailsApplication.config.portal.contactEmail
-
-        mailService.sendMail {
-            async true
-            to params.email.to
-            subject params.email.subject
-            body(
-                view: "/wps/${params.email.template}",
-                model: params
+        params.portalConfig = grailsApplication.config.portal
+
+        if (grailsApplication.config.grails.mail.disabled != true) {
+            mailService.sendMail {
+                async true
+                to params.email.to
+                subject params.email.subject
+                body(
+                    view: "/wps/${params.email.template}",
+                    model: params
+                )
+            }
+        }
+        else {
+            // email is disabled for dev's
+            println (
```

Comment:
Perhaps use a logging statement here instead so output can be controlled by logging framework like other logging statements.